packages/trader/test/index.ts

```diff
@@ -0,0 +1,23 @@
+import { run } from "../src/index";
+import { web3, assert } from "hardhat";
+
+describe("index.js", function() {
```

Comment:
This is a dumb sample test just to provide and example and prove the setup works.